sabnzbd/misc.py

```diff
@@ -68,12 +68,17 @@
 RE_UNITS = re.compile(r"(\d+\.*\d*)\s*([KMGTP]?)", re.I)
 RE_VERSION = re.compile(r"(\d+)\.(\d+)\.(\d+)([a-zA-Z]*)(\d*)")
 RE_SAMPLE = re.compile(r"((^|[\W_])(sample|proof))", re.I)  # something-sample or something-proof
+RE_SAMPLE_EXTENSIONS = re.compile(
+    r"^(\.mkv|\.avi|\.divx|\.xvid|\.mov|\.wmv|\.mp4|\.mpg|\.mpeg|\.ogg|\.jpg|\.jpeg|\.png|\.par2)$", re.I
```

Comment:
list of media files (does not include metadata files like nfo/png/jpg   or things like subtitles):
https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/MediaFiles/MediaFileExtensions.cs

subtitles:
https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/Extras/Subtitles/SubtitleFileExtensions.cs

can see this list can get pretty long..